db/migrate/20220429001408_add_users_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] <b>REQUIRED:</b> The same goes here, Kindly make sure that the foreign key for the user in the likes table is named `author_id` instead of `user_id`.

Reply:
Hi thanks for the review, Kindly ask you to share with me the link of how to change this because following the milestone resources they teach to do it like that